.env.dev-sample

```diff
@@ -21,3 +23,10 @@ KAFKA_PASSWORD="strong422example.!_password,463qteuayrdfghfdg"
 ### Endpoint
 ALLOWED_IPS="*"  # Set "*" for all, otherwise comma separated list of IP addresses
 ENDPOINT_PATH="sensornode"  # Without leading slash
+
+### Postgres
+POSTGRES_DB=heldatapipe
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
```

Comment:
Never use easily guessable password for any user even in docker

Reply:
This has been pretty much the standard approach for the development in all of the projects.